setup.cfg

```diff
@@ -1,59 +1,65 @@
-[build_sphinx]
-source-dir = docs
-build-dir = docs/_build
-all_files = 1
-
-[build_docs]
-source-dir = docs
-build-dir = docs/_build
-all_files = 1
-
-[upload_docs]
-upload-dir = docs/_build/html
-show-response = 1
-
 [tools:pytest]
-minversion = 2.2
```

Comment:
keeping a minversion is OK I think (but bump it to something newer)

Reply:
Is 6.0 too new?

Reply:
No, I think 6.0 is a good number.

Reply:
6.0 it is.